sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - SIG Docs put meaningful effort into growing its contributor and reviewer base in 2021, introducing a shadow program for PR Wrangling as well as dedicating more time to being active via our Slack community channel. This is an ongoing effort to grow our contributor base to a stable number, alongside easing the burden on our small group of reviewers and approvers.
+   - Alongside growing our contributor base, SIG Docs also worked on a leadership transition strategy to bring community members into leadership roles. Via a specialized six month mentorship program expertly led by Steering Committee member Paris Pittman, SIG Docs was able to grow its leadership cohort for the main SIG, as well as some of its subgroups, adding new co-chairs and tech leads.
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
```

Comment:
would another one be the consistent maintenance of the docset and strategies therein on how to make that happen? what about some other initiatives that I see on your agendas? do you have project board for the group or any subprojects?

Reply:
We don't have a project board, to the best of my knowledge. Ccing the other leads/chairs on here so that they can provide further insight.
@jimangel @sftim @onlydole @kbhawkey @reylejano @a-mccarthy @natalisucks @tengqm

Reply:
There is a project board for the blog subproject
https://github.com/kubernetes/website/projects/11